Documentation/DataSources/DataSources.md

```diff
@@ -9,17 +9,17 @@ Workbooks support these data sources:
 * [Metrics](#metrics)
 * [Azure Resource Graph](#azure-resource-graph)
 * [Alerts](#alerts)
+* [Custom Endpoint](#custom-endpoint-(preview))
+* [Azure Resource Manager](#azure-resource-manager-(preview))
 * [Workload Health](#workload-health)
 * [Azure Resource Health](#azure-resource-health)
-* [Custom Endpoint](#custom-endpoint)
 * [Azure Data Explorer](#azure-data-explorer)
-* [Azure Resource Manager](#azure-resource-manager)
 * JSON
 * Custom Provider
 
 You can also use the [Merge](#merge-data-from-different-sources) option in the query control to combine data from different data sources. 
 
-## Logs
+### Logs
```

Comment:
Why did you go with H3s? H2s seem right from a hierarchy perspective. No big deal - just curious.

Reply:
When I was trying to make some of the broken links work I thought changing the H2 to H3 would fix it but it doesn't. Changing it back.